search-app/src/cmr/search/results_handlers/metadata_results_handler.clj

```diff
@@ -109,6 +111,13 @@
                                (when data
                                  (x/element :data {} (json/generate-string data))))))))]))
 
+(defn xml-escape-umm-json-metadata
```

Comment:
Nitpick: this name is a little confusing. Would something like `escape-special-characters-for-umm-json` make sense?

Reply:
`escape-special-characters-for-umm-json` is not clear on what the escaping is for. Is it for url escaping or xml escaping? That's why I want the `xml` in there.